### DISCUSSION TITLE
好起来了，不用答题解bl锁了

### DISCUSSION THREAD

**Discussion by inchacha** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#204] [General]
禁止国行版刷国际版，那答个鸡毛，直接买港澳版
![IMG_20240711_000950](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/124475315/9589bf9a-329b-4adb-9c50-45adf015a249)

**Comment by Originalimoc** [comment]
我开始怀疑是不是有内鬼，慢慢毁掉小米的内鬼
